### ISSUE TITLE
Lone Druid and How to Fix Him

### ISSUE THREAD

**Issue by Ynscription** [OpenAngelArena/oaa#1314]
So there are several issues that feel frustrating/useless/_unbearable_ about LD, here are the ones i have recollected and how to fix them. This are ideas that may need tinkering.

1. **Boots**: The most _unbearable_ issue with this heroe is the fact that you must choose between having good boots on the bear or on the druid, since only one farming core is dropped from bosses per player.
**Fixes**:
	- The less elegant and more complex: Have the bear count as a player when dropping the farming core. This could result in 2 farming boots on the bear or druid, so either lock the item to each one so it can **NOT** be used by the other, or create the **Unique Farming Modifier**,wich would only allow the farming component of one set of boots to be active on a hero at any given moment.
	- The more elegant (that may or may not benefit other heroes) add upgraded boots without the farming component. These give the same benefits (attack damage, stats, move speed...) of the upgraded farming boots but without the farming component. This could allow for interesting new items, like adding the farming core to a battlefury to make a superbattlefury (just a concept).

2. **Reflex Core**: The same issue as with boots, but it **does NOT need fixing**, since usually you only get the survivability on the druid and not on the bear, and you can still get a bkb.

3. **Duels**: There are a lot of frustrating interactions between the bear and the duel. Now, _bear_ with me for a moment. In regular dota you really treat the bear as another hero when playing LD and you go into team fights with the bear summoned and the ability off cooldown, so that if the bear dies you can summon another one.
**Fix**: Treat the bear as a hero in terms of duel interactions: saving its state before the duel and resetting it afterwards.  By doing this, it would emulate this behaviour/mechanic/gimmick in the duels, (Without a refresher item) you would get two bears on the duel since the ability is off cooldown.

4. **Cores**: Since LD effectively has 12 inventory slots, he needs twice as many cores as everybody else. This is sort of an issue right now (not really since a good team would help you and focus on bosses a lot) but it forces the allies to work for the LD in order to make him work, which is unrealistic for a pub custom game match. 
**Fix**: Implement Azazel. With the great farming capabilities of LD, once azazel is implemented (the guy that sells cores) this would no longer be an issue.

5. **Demolish**: As it stands right now, this ability is not very usefull since there are no towers in OAA (although the magic resistance is nice) however since LD needs so many cores i would sugest the following fix, that would synergize with this need.
**Fix**: Change the **[Building Attack Damage Bonus: 40%]** to **[Boss Attack Damage Bonus: 40%]** (The 40% could probably use some tinkering as it sounds rather broken on paper)

**Comment by warpdragon** [comment]
Fantastic post! Thank you!

**Comment by hallelsegel** [comment]
The points made in 4 and 5 seem very good, they fix the problem and aren't too complicated. If easily implemented 3 is good too, I feel like 1 needs more thought cuz damn those fixes are drastic. Good job!

**Comment by Chronophylos** [comment]
Thanks this is what I've waited for, a good post with not only a problem but also a solution!

**Comment by Haganeko** [comment]
A lot of these were under discussion already, but it's nice to see them all collected like this.
IMO the boots will be the greatest problem, since non farming boots may confuse new players.
The bear already gets teleported into the duel. The  only thing is that the duel does not save its state, but that happens with all summons.
About the cores: having NGP cores again would help with that too. Recall that Azazel is tier 5 - he wouldn't be beaten early at all.

**Comment by Ynscription** [comment]
@Haganeko & @hallelsegel 
I agree that the boots issue is a brainsqueezer, but there has to be a way to allow LD to have good boots on both, otherwise it feels like you are wasting an inv slot. Also one of this could be implemented as a temporary fix while a better solutions is found.

@Haganeko 
As far as i know (last played LD on tuesday irc) the bear doesn't get teleported to the arena, you have to use its Q, or summon it during the duel. 

While you could argue its a summon, it's really meant to be played like a hero. Also, as I explained on point 3 there is the whole mechanic/gimmick of the double bear. Additionally if LD dies during the duel, once the duel finishes he gets no bear outside the duel, which in the late game can be fine, but in the first duel is devastating (could I say _**unbearable**_).

About the cores: I don't know what the first sentence means at all, I don't know what NGP means specifically. 
The Azazel fix is really just speculation, until we get Azazel there is no way of telling if that will fix the core shortage issue. However I beliebe it could.

**Comment by Haganeko** [comment]
All player controlled units supposedly go to duel: I know for a fact that it works with Chen and Enchantress creeps.

**Comment by Ynscription** [comment]
@Haganeko well, it doesn't work on the bear.

**Comment by Minnakht** [comment]
Lateral solution to the boot problem: In OAA, there's two extra skill ranks. How about making the bear get enough movement speed at rank 5 to make boots unnecessary? Then you could just get six normal items on it if you really wanted to.

**Comment by Ynscription** [comment]
@Minnakht
Thats a very good idea, but phase boots are a must on the bear for the phase shift. I would either put the extra movespeed on the druid or give the choice to the player (have one talent be for the bear and one for the druid at the same lvl so the player can choose who gets the movespeed).

**Comment by hallelsegel** [comment]
I was going to suggest making bear eat brown boots for movement speed, but Minna's is so much simpler lol. Not a huge LD player myself but is phase really so important on the bear?

**Comment by Haganeko** [comment]
Bear likes to chase people to use the Entangle. Also druid and bear dislike being stopped by each other.

**Comment by SquawkyArctangent** [comment]
Actually, the move speed on the bear would make perfect sense. Boots are usually one of the last things to upgrade anyway, so you could just stick tier 1 Phase on the bear just for the active and just focus on all the other items. Bear likes Radiance for farming anyway, so it's not like not getting the farming components on it really matters a whole bunch.

**Comment by Minnakht** [comment]
dunno, man, sounds like you're some kind of micro gods to repeatedly use the phase active on the bear. I wouldn't have thought of it.

...perma passive phase bear at rank 6?

**Comment by Ynscription** [comment]
@hallelsegel it really is very important, like @ Haganeko says.

@SquawkyArctangent I usually upgrade boots ASAP to farm more, is like expanding early in starcraft. Also I would rather have farming phase boots than radiance and focus more on damsge on the bear. In OAA the power level is increased and the bear is not as tanky as in dota, so the radiance is not so good against enemies. But still there are a lot of different ways to play LD, thats why i think giving the choice to the player on the movespeed matter would be best, have the talent be either movespeed for bear or for druid.

@Minnakht it's not that hard, and the perma phase while it sounds nice you also want to be able to body block enemies with the bear so they dont kill the druid.

tl;dr: Since there is a variaty of ways to play LD I still think the best solution would be to give the players the choice through talents to give boot speed bonus to the bear or the druid.

**Comment by SquawkyArctangent** [comment]
Without implementing some sort of completely custom talent system, that'd require cannibalizing a current set of talents, so ... would you really rather +50 Druid Movement Speed or +50 Bear Movement Speed over -10 Savage Roar Cooldown or +1.5s Entangling Roots Duration?

**Comment by Ynscription** [comment]
@SquawkyArctangent I think implementing an additional talent for all heroes would be great for OAA (since the last ability upgrade is at lvl 50), but i don't see that happenning any time soon if at all.
If I had to choose I would put the movespeed talents on the lvl 15 or 20 and here is why:
In the early game you can just get normal boots and roll with those, so at lvl 10 seems too soon. The lvl 25 talents are really good specially the entangle one. However 50 DMG in OAA is not that big of a deal, so the lvl 15 talents seem like very replaceable, The lvl 20 talents are really good as well, so i would rather not get rid of those, but in terms of timing lvl 20 might be a good time to get the movespeed talent. 

I would have to play a couple of games to see when it would be better to get the talents. Perhaps another alternative would be to move the lvl 20 talents to lvl 15, the 25 to 20 and then add the movespeed to lvl 25 but that could be incredibly broken, I dont know just a concept.

**tl;dr**: I would substitute the lvl 15 talents for the movespeed.

**Comment by Haganeko** [comment]
We *could* just make sure our farming boots don't stack and give Druid two (which I don't like much).
Could also give the bear a skill on ability level five which passively gives it MS and activates to give it phase.

**Comment by Ynscription** [comment]
@Haganeko The first idea is one I exposed in point 1 of my original post, and i think would work although not elegant. 
However this phase boots ability substitute sounds super easy to implement and exactly what would fix the whole issue altogether. I second this idea very strongly.

**tl;dr**: I think giving the bear an ability that works like phase boots is the best idea so far.

**Comment by Ynscription** [comment]
I just played LD and can confirm that the bear still doesn't get teleported to the duel arena like other summons. Should I make a separate issue for this?

**Comment by Haganeko** [comment]
Actually a suggestion. Closed.
